org.linux_hardware.hw-probe.yaml

```diff
@@ -0,0 +1,393 @@
+app-id: org.linux_hardware.hw-probe
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: 1.6
+sdk: org.freedesktop.Sdk
+command: /app/usr/bin/hw-probe-flatpak
```

Comment:
You can only set `hw-probe-flatpak`

Reply:
`flatpak-builder` is failed with the following error in this case:

    Error: Command 'hw-probe-flatpak' not found

Reply:
Install the binary in `/app` prefix instead of `/app/usr`.

Reply:
This is fixed by one of the previous commits.

For now the script doesn't install anything to `/app/usr/`. As a side effect this have fixed strange icon problem.